modules/games/definitions/jesonmor/instructions.ts

```diff
@@ -0,0 +1,18 @@
+import { JesonmorDefinition } from "./_types";
+
+const jesonmorInstructions: JesonmorDefinition["instructions"] = {
+  startTurn: { line: ["Go!"] },
+  selectunit: { line: ["Select", "where to move", { unitat: "selectunit" }] },
+  selectmovetarget: {
+    line: [
+      "Press",
+      "move",
+      "to move",
+      { unitat: "selectunit" },
+      "to",
+      "selectmovetarget"
```

Comment:
I've tried to also make the dynamic instruction show the effect of the move, for example if we kill something! So something like this:

```suggestion
      "to make",
      { unitat: "selectunit" },
      {
        ifelse: [
          { anyat: ["units", "selectmovetarget"] },
          { line: ["stomp", { unitat: "selectmovetarget" }] },
          { line: ["go to", "selectmovetarget"] }  
        ]
      }
```